Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Lighthouse"
 uuid = "ac2c24cd-07f0-4848-96b2-1b82c3ea0e59"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.14.3"
+version = "0.14.4"
```

Comment:
Technically, this does add an extra field to the output of `binary_statistics`, but not sure this qualifies as a breaking change.